core/src/com/unciv/logic/multiplayer/storage/SimpleHttp.kt

```diff
@@ -67,4 +80,20 @@ object SimpleHttp {
             return socket.localAddress.hostAddress
         }
     }
+
+    /** Builds an URL out of the provided components, automatically joining them with a `/`. The components are only allowed to have no slashes or one slash at the end. */
+    fun buildURL(vararg components: String): String {
+        val joined = components.reduce { acc, cur ->
+            if (acc.endsWith('/')) {
+                acc + cur
+            } else {
+                acc + "/" + cur
+            }
+        }
+        return if (joined.endsWith('/')) {
+            joined.substring(0, joined.length - 1)
+        } else {
+            joined
+        }
```

Comment:
I think there's a removeSuffix function in string, so this could just be joined.removeSuffix("/")